plans/convert.pp

```diff
@@ -56,6 +56,12 @@
   )
 
   out::message('# Gathering information')
+  $failed_node_counter = peadm::check_availability($pe_installer_targets)
+  if $failed_node_counter > 0 {
```

Comment:
If the previous advice is implemented then this check is not required. `peadm::check_availability()` will fail the plan with proper message and details